magenta/models/arbitrary_image_stylization/arbitrary_image_stylization_build_mobilenet_model.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Methods for building real-time arbitrary image stylization model
+using MobileNetV2
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+from magenta.models.arbitrary_image_stylization import arbitrary_image_stylization_losses as losses
+from magenta.models.arbitrary_image_stylization import nza_model as transformer_model
+from magenta.models.image_stylization import ops
+
+try:
+  from nets.mobilenet import mobilenet_v2, mobilenet
+except ImportError:
+  print("Cannot import MobileNet model. Make sure to install slim "
+        "models library described "
+        "in https://github.com/tensorflow/models/tree/master/research/slim")
+  raise
+
+slim = tf.contrib.slim
+
+
+def build_mobilenet_model(content_input_,
+                          style_input_,
+                          mobilenet_trainable=True,
+                          style_params_trainable=False,
+                          transformer_trainable=False,
+                          reuse=None,
+                          mobilenet_end_point='layer_19',
+                          style_prediction_bottleneck=100,
+                          adds_losses=True,
+                          content_weights=None,
+                          style_weights=None,
+                          total_variation_weight=None):
```

Comment:
For functions defined in this module, can you add docstrings explaining what each argument is for? For instance, it's not immediately clear what behavior to expect if `style_prediction_bottleneck <= 0`, which judging from line 118 is supported.